### PR TITLE
improve "open in app" page

### LINKED ISSUE #6119: Unclear UX/Journey for some App users
**Is your feature request related to a problem you are facing?**
Users clicking on the links from our on-premise instance are (for a few) confused by the app landing page ("Download the app" or **Continue to the app**)

Not a huge deal, but

**Describe the solution you'd like**
I have edited the `/usr/share/jitsi-meet/libs/app.bundle.min.js` to "Continue to the **Meeting**" and it solved the issue to some extent (even though the lines "Download" and "Continue" might be better switched in position ("Continue" an "Download" bellow) in terms of natural flow ?

**Describe alternatives you've considered**
Maybe we could add *some* content to explain the difference.

### PR DESCRIPTION
**Pull request by obfusk**
Fixes #6119.

### CODE REVIEW

**Comment by obfusk** [comment]
Screenshot:

Screenshot from 2020-05-14 21-33-02 cleaned

File: react/features/deep-linking/components/DeepLinkingMobilePage.web.js [hunk 1]
```diff
@@ -129,6 +129,21 @@ class DeepLinkingMobilePage extends Component<Props> {
...
                     </p>
+                    <p className = { `${_SNS}__text` }>
+                        { t(`${_TNS}.ifHaveApp`) }
+                    </p>
+                    <a
+                        { ...onOpenLinkProperties }
+                        className = { `${_SNS}__href` }
+                        href = { generateDeepLinkingURL() }
+                        onClick = { this._onOpenApp }>
+                        <button className = { `${_SNS}__button ${_SNS}__button_primary` }>
```
Comment (luixxiul) [inline react/features/deep-linking/components/DeepLinkingMobilePage.web.js:140]: @jitsi/jitsi-meet  I am trying to add a button element for a possible PR for https://github.com/jitsi/jitsi-meet/issues/5992, and I am wondering if react modules such as `Button` on `atlaskit/button` ([example](https://github.com/jitsi/jitsi-meet/blob/72137a28114ca19000f2d52e2f6cf8df570b29a0/react/features/settings/components/web/ProfileTab.js#L3)) is encouraged or not. Does it depend on each case?

**Comment by saghul** [comment]
@jallamsetty1 maybe you can PTAL since you'll be working in this component soon?

**Comment by jallamsetty1** [comment]
@obfusk, thank you for the PR. It LGTM ! Can you please resolve the conflict so that we can merge it, thanks !

**Comment by obfusk** [comment]
@jallamsetty1 conflict resolved.

**Comment by jallamsetty1** [comment]
Jenkins please test this please.

**Pull request merged**